web/src/main/java/org/apache/shiro/web/env/IniWebEnvironment.java

```diff
@@ -263,20 +263,13 @@ protected FilterChainResolver createFilterChainResolver() {
         Ini ini = getIni();
 
         if (!CollectionUtils.isEmpty(ini)) {
-            //only create a resolver if the 'filters' or 'urls' sections are defined:
```

Comment:
I'd like to see a comment here, if you don't mind:
`// SHIRO-610: Allow creation of FilterChainResolver even without urls and filters sections.`

Reply:
> I'd like to see a comment here, if you don't mind:
> `// SHIRO-610: Allow creation of FilterChainResolver even without urls and filters sections.`

@bmhm I'm not a big fan to add the jira related informations, we already have the git commit history for this.

Reply:
Alright. I was requesting this because we had security-related code changes which could have been prevented by such comments. Why? Because  you do not always look into the history before accepting code or changing code.

Reply:
@bmhm ok, so may be adding the related CVE id?